packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -342,6 +346,16 @@ describe('Connection string', function () {
         .getText();
       expect(errorMessage).to.equal('Authentication failed.');
     }
+
+    // for multiple connections click the review button in the toast
```

Comment:
There was a bullet point in the ticket to test this and I realised I could just tack it onto an existing test.